cpg_workflows/metamist.py

```diff
@@ -635,11 +695,8 @@ def parse_reads(  # pylint: disable=too-many-return-statements
         index_location = None
         if reads_data[0].get('secondaryFiles'):
             index_location = reads_data[0]['secondaryFiles'][0]['location']
-            if (
-                location.endswith('.cram')
-                and not index_location.endswith('.crai')
-                or location.endswith('.bai')
-                and not index_location.endswith('.bai')
+            if (location.endswith('.cram') and not index_location.endswith('.crai')) or (
+                location.endswith('.bai') and not index_location.endswith('.bai')
```

Comment:
```suggestion
                location.endswith('.bam') and not index_location.endswith('.bai')
```

It looks like the original version of this contained a typo, but we never found out because we don't use BAMs

Reply:
Updated